.github/workflows/linter.yml

```diff
@@ -1,33 +1,30 @@
 ---
-name: Lint Project
+name: Lint
 
 on:
-  pull_request:
-    branches: [master, main]
+  push: null
+  pull_request: null
 
 jobs:
   build:
-    name: Lint Code Base
+    name: Lint
     runs-on: ubuntu-latest
 
     permissions:
       contents: read
       packages: read
-      # To report GitHub Actions status checks
       statuses: write
 
     steps:
-      - name: Checkout Code
+      - name: Checkout code
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
-      - name: Lint Code
-        uses: super-linter/super-linter/slim@v5
+      - name: Super-linter
+        uses: super-linter/super-linter@v6.2.0
```

Comment:
Why are we dropping the slim version of the linter for the larger one?

Reply:
My bad, i forgot to swap back to the slim version after we decided to not check the CSS